### PR TITLE
Installation manual

### PR DESCRIPTION
**Pull request by davi17g**

### CODE REVIEW

**Comment by roimenashe** [comment]
Not sure if its open for review yet but `yarn build` fails with many `Module not found: Error: Can't resolve 'raw-loader' in ...` errors

File: docs/gettingstarted/install/docker-compose.md [hunk 1]
```diff
@@ -21,42 +21,34 @@ Before you start please make sure all the relevant IP/DNS addresses and ports ar
...
 
 :::tip
-**Note:** make sure you have [AeroLab](https://github.com/aerospike/aerolab/tags) installed locally and clone our repo to utilize our installation scripts.
+**Note:**  clone our repo to utilize our installation scripts.
 ``` shell
 git clone https://github.com/aerospike/aerospike-database-recovery.git 
 ```
 :::
 
 ### Configure license key
```
Comment (hev) [inline docs/gettingstarted/install/docker-compose.md:33]:
```suggestion
### Configure license key
```

```suggestion
### Download license key
```

File: docs/gettingstarted/install/docker-compose.md [hunk 2]
```diff
@@ -21,42 +21,34 @@ Before you start please make sure all the relevant IP/DNS addresses and ports ar
...
+In order for the system to work, you need an Aerospike Enterprise License, with the database-recovery entitlement set to true.
```
Comment (hev) [inline docs/gettingstarted/install/docker-compose.md:34]:
```suggestion
In order for the system to work, you need an Aerospike Enterprise License, with the database-recovery entitlement set to true. Once you have completed the Alpha agreement your license key will be available on the support portal. 
```

File: docs/gettingstarted/install/docker-compose.md [hunk 3]
```diff
@@ -21,42 +21,34 @@ Before you start please make sure all the relevant IP/DNS addresses and ports ar
...
-```
-
-### Deploy using AeroLab and Docker Compose
+### Deploy Docker Compose
 In the `/scripts/docker-compose` folder you can use our script to create the necessary components of our [architecture](../../architecture.md). 
 
 This includes:
 
-1. [Source Cluster](../setup/xdr.md) with static XDR Configuration required for the [sample policies in our setup guide](../setup/adr.md). 
+1. [Source Cluster](../configuration/source-cluster/config.mdx) with static XDR Configuration required for the [sample policies in our adr entities guide](../configuration/adr-entities/config.mdx).
```
Comment (hev) [inline docs/gettingstarted/install/docker-compose.md:41]: I found the following confusing as it relates to the install. 

1. Which backup-conf file to use which in this case should be static-xdr.conf. I vote we do not include the other examples. 
2. We should include a sample aerospike.conf that is typical for docker on a localhost in that config folder.
3. IMO this makes the command much more copy/pastable. I also say we remove the cd step of the command and rely on the user to navigate to the directory before copy/pasting.

**Review by hev — CHANGES_REQUESTED** [review]
So far I have only had a chance to review docker-compose instructions. I left some specific feedback but I also hit the following error when I tried to create the cluster connection object. 

```
➜  docker-compose git:(installation-manual) ✗ curl -i --location --request POST "localhost:8080/v1/connect/create" \
--header 'Content-Type: application/json' \
--data-raw '{
    "srcClusterName": "srcCluster1",
    "clusterRoles": ["SHIPPING", "RESTORE"],
    "srcClusterHost": "host.docker.internal",
    "srcClusterPort": 3000,
    "srcClusterUser": "tester",
    "srcClusterPwd": "psw",
    "backupDCName": "AdrDC",
    "smdPolicy": {
        "duration": 86400
    },
    "smdLastExecuted": 0
}'
HTTP/1.1 500
Content-Type: text/plain;charset=UTF-8
Content-Length: 55
Date: Mon, 11 Sep 2023 16:29:50 GMT
Connection: close

Error 20: Namespace not found in partition map: catalog%
```

**Comment by hev** [comment]
@davi17g I am still getting this error:

```
➜  docker-compose git:(installation-manual) ✗ curl -i --location --request POST "localhost:8080/v1/connect/create" \
--header 'Content-Type: application/json' \
--data-raw '{
    "srcClusterName": "srcCluster1",
    "clusterRoles": ["SHIPPING", "RESTORE"],
    "srcClusterHost": "source-cluster",
    "srcClusterPort": 3000,
    "srcClusterUser": "tester",
    "srcClusterPwd": "psw",
    "backupDCName": "AdrDC",
    "smdPolicy": {
        "duration": 86400
    },
    "smdLastExecuted": 0
}'
HTTP/1.1 500
Content-Type: text/plain;charset=UTF-8
Content-Length: 55
Date: Wed, 13 Sep 2023 20:38:10 GMT
Connection: close

Error 20: Namespace not found in partition map: catalog%
```

**Comment by hev** [comment]
Note, I pushed up a few changes/improvements.

1. Removed /scripts/docker-compose/config/backup.conf - not sure what this one is for
2. Added /scripts/docker-compose/config/aerospike.conf - for the source cluster (super basic)
3. Fixed a typo 'source-cofig` in the script and commands
4. Some wording and updates related to my previous comments here.

**Comment by hev** [comment]
Got this working with David.

**Review by hev — APPROVED** [review]

**Comment by roimenashe** [comment]
@reugn @davi17g 
Configuration files are not formatted, indentation (spaces) are not aligned and sometimes we have line brakes before stanzas (between namespace definitions) and sometime don't.
We also have different memory config for different namespaces (first ones get 4G and the others get 1G not sure if we want to use that in our examples).

Other than that looks good just need to verify that what we include in our docs actually works, if SA can try to install and configure it - it will be great.

**Pull request merged**